scripts/run_mteb_german.py

```diff
@@ -27,7 +27,7 @@
     "TenKGnadClusteringS2S",
 ]
 
-TASK_LIST_PAIR_CLASSIFICATION = ["PawsX"]
+TASK_LIST_PAIR_CLASSIFICATION = ["FalseFriendsGermanEnglish", "PawsX"]
```

Comment:
```suggestion
TASK_LIST_PAIR_CLASSIFICATION = ["PawsX"]
```

@Muennighoff I assume how do we go about updating benchmarks?

Reply:
I def. believe it belongs on the benchmark! We simply do not want to outdate all previous results trained on the german MTEB, which this action would do as previous models are not evaluated on this task. Thus I would rather make larger amendments to the benchmarks, rather than a dataset at a time.

Reply:
Ah get it. Yeah makes sense, if you want me to I could also do the evaluation for the open source / access models fitting on a 3090.

Reply:
I think this addition is fine!

Only maybe for everything that already has a leaderboard tab (i.e. English, Chinese, Polish, French), let's decide on additions after all datasets have been added!

Reply:
Actually just checked MTEB and seems like German does not have an overall tab (only for clustering it seems). @Muennighoff does that mean that we can add it without issue? (otherwise, it seems like all models run on the German tasks models can be run on less than 1gb of memory with the exception of the [sentence-t5-xxl](https://huggingface.co/sentence-transformers/sentence-t5-xxl))

Reply:
Yeah that's what I meant! As German has no Overall tab yet, let's just add it!

Reply:
yep, seems like we posted at the same time (I at least didn't see the comment above before after)